doc/source/whatsnew/v2.0.0.rst

```diff
@@ -465,7 +465,9 @@ The arguments signature is similar, albeit ``col_space`` has been removed since
 it is ignored by LaTeX engines. This render engine also requires ``jinja2`` as a
 dependency which needs to be installed, since rendering is based upon jinja2 templates.
 
-The pandas options below are no longer used and will be removed in future releases.
+The pandas latex options below are no longer used and have been removed. The generic
+max rows and columns arguments remain but for this functionality should be replaced
```

Comment:
Why do we need to keep the `max_rows`, `max_columns`?

Reply:
becuase they still impact the other DataFrame rendering methods, like to_string, and to_html (under the HTMLFormatter and not Styler)

Reply:
Ah got it. Thanks!